x-pack/plugins/cases/public/components/all_cases/columns_popover.test.tsx

```diff
@@ -14,15 +14,22 @@ import type { AppMockRenderer } from '../../common/mock';
 import { createAppMockRenderer } from '../../common/mock';
 import { ColumnsPopover } from './columns_popover';
 
-// FLAKY: https://github.com/elastic/kibana/issues/174682
-describe.skip('ColumnsPopover', () => {
+describe('ColumnsPopover', () => {
   let appMockRenderer: AppMockRenderer;
 
   beforeEach(() => {
     appMockRenderer = createAppMockRenderer();
     jest.clearAllMocks();
   });
 
+  afterEach(() => {
+    appMockRenderer.queryClient.getQueryCache().clear();
```

Comment:
Would be nice to have just one appMockRenderer.cleanup function that does everything. If there is something to change there would be just one place to update, wdyt?

Reply:
Updated in [904fb11](https://github.com/elastic/kibana/pull/191212/commits/904fb1165e6eb718d356d7fc1fd05a3b63764789)